Doc/library/dis.rst

```diff
@@ -93,6 +93,9 @@ code.
       Return a formatted multi-line string with detailed information about the
       code object, like :func:`code_info`.
 
+   .. versionchanged:: 3.7
+      This can now handle asynchronous generator and coroutine objects.
```

Comment:
Minor nit: "coroutine and asynchronous generator" is slightly easier to parse when reading (since the scope of the adjective is unambiguous)